checksec.cpp

```diff
@@ -60,6 +60,13 @@ Checksec::Checksec(std::string filepath) : filepath_(filepath), loadedImage_(fil
         auto size = std::min(loadConfigData.size(), sizeof(loadConfig));
         memcpy(&loadConfig, loadConfigData.data(), size);
         loadConfigSize_ = loadConfigData.size();
+        if ((loadConfig.Size > loadConfigSize_) && (loadConfig.Size <= size)) {
+            std::cerr << "Warn: load config larger than reported by data directory entry,"
+                      << " overwriting"
+                      << "\n";
+            memcpy(&loadConfig, loadConfigData.data(), loadConfig.Size);
```

Comment:
We probably need a bounds check on `loadConfigData.data() + loadConfig.Size`, right? 

LGTM otherwise!

Reply:
Don't think so, since the total data size is `loadConfig.Size`, we just overwrite the data we already copies earlier, not add to it.

Reply:
I meant for the PE container itself -- I could be wrong, but I think someone could modify a PE to set the load config's size to a size larger than the PE itself, meaning that `loadConfigData.data() + loadConfig.Size` would point past the end of the PE in memory.

Reply:
Ah, I see what you mean. Yeah, I'll add that check!